src/applications/personalization/profile/tests/components/alerts/Profile.data-unavailable-error.unit.spec.js

```diff
@@ -144,16 +140,40 @@ describe('Profile "Not all data available" error', () => {
     expect(view.queryByTestId(ALERT_ID)).not.to.exist;
   });
 
-  it('should not be shown if there is a 500 error with the `GET service_history` endpoint and the user is not a vet', async () => {
-    server.use(...mocks.getServiceHistory500);
+  context('when veteranStatus is null', () => {
```

Comment:
These tests for when the `veteranStatus` is `null` already existed, I just refactored things a bit